src/engine/input/key_mapping/InputGroupList.cs

```diff
@@ -178,6 +178,11 @@ public void InitGroupList()
         EnsureNavigationFlowIsCorrect();
     }
 
+    public void ClearGroupList()
+    {
+        this.FreeChildren();
```

Comment:
Related to my comment on the issue, for example `activeInputGroupList` is something that immediately pops out to me as a potential place where there's probably stored Controls that should be cleared.

Reply:
Thank you for the advice! 

I looked over my code, and the error logs in more detail. I think I found and clear references to children that were freed. 

I am now getting the correct behavior, and no longer getting the error from before.